minetest.conf.example

```diff
@@ -74,6 +74,10 @@
 #    type: float min: 0.001 max: 10
 # mouse_sensitivity = 0.2
 
+#    ADS mouse sensitivity multiplier.
```

Comment:
FWIW, this file is updated automatically by a script in builtin, based on the settingtypes.txt file. You can change it if you want, but it will be updated later on automatically.